proprietary/tokens/src/components/ams/avatar.tokens.json

```diff
@@ -0,0 +1,48 @@
+{
+  "ams": {
+    "avatar": {
+      "border-width": { "value": "{ams.border-width.md}" },
+      "border-style": { "value": "solid" },
+      "font-family": { "value": "{ams.text.font-family}" },
+      "font-size": { "value": "{ams.text.level.6.font-size}" },
```

Comment:
Since the line height determines the size of the Avatar, does this level align with menu items in the Header? Maybe we need an example story?

Reply:
I'll add an example story in the next commit.

Reply:
Example story added.